idx/cassandra/cassandra.go

```diff
@@ -450,6 +451,13 @@ func (c *CasIdx) Delete(orgId uint32, pattern string) ([]idx.Archive, error) {
 		}
 	}
 	statDeleteDuration.Value(time.Since(pre))
+
+	// there is nothing higher up in the call path that uses MetricDefinitions
+	// so this is the safest place to release the objects in MetricDefinition
+	// that have been interned
+	for _, arc := range defs {
+		idx.InternReleaseMetricDefinition(arc.MetricDefinition)
+	}
```

Comment:
same remark here as for bigtable's Delete()

Reply:
Same comment.